### ISSUE TITLE
Support python-idb loader

### ISSUE THREAD

**Issue by XVilka** [BinaryAnalysisPlatform/bap#859]
To be able to open IDB files without IDA installed. Very useful for automation purposes.

- https://github.com/williballenthin/python-idb
- https://pypi.org/project/python-idb/

**Comment by ivg** [comment]
Yep, I'm following this project. It's pretty low-level and my main concern is the stability of the representation between different versions of IDA. So far, we're spawning an IDA process and use ida-python to extract useful information, works pretty well.

**Comment by XVilka** [comment]
> The idea, is all these three modules should be factored out from the `bap-ida-python` repository, probably into `bap-python` (we then can submodule `bap-python` into `bap-ida-python` to make it available again in IDA).

from https://github.com/BinaryAnalysisPlatform/bap/issues/929#issuecomment-471061600 could be helpful in this case as well.
It will allow also the better automated testing of such integration.

**Comment by XVilka** [comment]
The newer version of [python-idb 0.6.0 was released](https://github.com/williballenthin/python-idb/releases/tag/v0.6.0) today. And the support for [types section parsing is coming](https://github.com/williballenthin/python-idb/pull/82). If you have any particular requirement for python-idb please let me know.

**Comment by XVilka** [comment]
Now it supports all 6.x - 7.x version and can extract functions, types. Work for supporting 5.x versions and more information extraction is under way: https://github.com/williballenthin/python-idb/milestone/1 
It will be the part of 0.7.0 release.
At this point, I think, everything that is required for running BAP IDAPython is ready to work with python-idb, at minimum for running automated tests.
